bpdv3/Assets/Scripts/Player/PlayerDestination.cs

```diff
@@ -61,7 +62,6 @@ private void CheckBoundaries(Collider other)
             {
                 IsDestinationObtained = false;
                 _playerMovement.IsPlayerInputDetected = false;
-                other.gameObject.GetComponentInParent<TileProperties>().OccupiedDecreased();
```

Comment:
Because the tiles aren't using accupancy number for enemymovement now. It will not need to make use of this variable. Everything should still behave the same as before. It was just to change the value for a tile.